src/keymaps/keymaps.am

```diff
@@ -3,6 +3,8 @@ KEYMAP_FILES = \
     $(srcdir)/%reldir%/de-keymap.json \
     $(srcdir)/%reldir%/default-keymap.json \
     $(srcdir)/%reldir%/fr-keymap.json \
+    $(srcdir)/%reldir%/sv-keymap.json \
+    $(srcdir)/%reldir%/svmac-keymap.json \
```

Comment:
can you remove this

Reply:
sure. Do you want me to remove both lines, or just the svmac one that I forgot about?

Reply:
Just svmac